microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsUtils.java

```diff
@@ -75,13 +77,17 @@ public static String getJaxRsPathValue(IAnnotatable annotatable) throws JavaMode
 	 */
 	public static String getJaxRsApplicationPathValue(IAnnotatable annotatable) throws JavaModelException {
 		IAnnotation annotationPath = getAnnotation(annotatable, JAVAX_WS_RS_APPLICATIONPATH_ANNOTATION);
+		if (annotationPath != null) {

```

Comment:
I will change the signature of getAnnotation in  `getAnnotation(IAnnotable annotable, String name...)`